src/mono/wasm/runtime/es6/dotnet.es6.lib.js

```diff
@@ -4,13 +4,9 @@
 
 "use strict";
 
-#if USE_PTHREADS
-const usePThreads = true;
-const isPThread = `ENVIRONMENT_IS_PTHREAD`;
-#else
-const usePThreads = false;
-const isPThread = `false`;
-#endif
+const monoWasmThreads = process.env.MonoWasmThreads == "true";
```

Comment:
This works because emscripten evaluates dotnet.es6.lib.js in node at build time, right?

Reply:
Yes